src/chain.h

```diff
@@ -320,6 +324,7 @@ class CDiskBlockIndex : public CBlockIndex
             READWRITE(VARINT(nDataPos));
         if (nStatus & BLOCK_HAVE_UNDO)
             READWRITE(VARINT(nUndoPos));
+        READWRITE(hashAnchor);
```

Comment:
What will happen if they don't launch with ``-reindex``? Can we detect that case and print a message telling them to do so?